src/model/inline-document.ts

```diff
@@ -45,6 +45,15 @@ export type AstNodeWithLanguage = {
   containingDocument: JavaScriptDocument,
 };
 
+export class ContainingDocumentBackreference extends ScannedFeature {
+  kinds = new Set(['containing-document-backreference']);
+  identifiers: Set<string>;
+  constructor(public readonly document: Document) {
+    super(undefined, undefined, undefined, undefined, undefined);
+    this.identifiers = new Set([...document.identifiers]);
```

Comment:
It's not clear to me why this should have the same identifiers as the container document. Can you clarify that, and add a comment?

Reply:
You know, I went and dropped the identifiers altogether, essentially because there should only ever be one of these backreference items in a given inline document so identifiers are extraneous.  I think I defaulted to cloning them at the time because I didn't know what else they should be; upon reflection, I decided they shouldn't be anything.